systemd/tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py37
+envlist =
+    {py27,py37}-{systemd}
+    flake8
+
+[testenv]
+usedevelop = true
+platform = linux|darwin|win32
```

Comment:
Are tests working on mac and windows? I can't make it work on my mac.

Reply:
No only Linux, as `systemd` is designed specially for it, it won't work on mac. All the tests are ran were on a Ubuntu VM.